### PR TITLE
Feat 75/manufacturing order page

### LINKED ISSUE #133: [Bug] Sidebar appears when we click on Forgot password in the login screen

### PR DESCRIPTION
**Pull request by tjosepo**
# Description

Take 3. Resolves #133.

## Type of change

Please delete options that are not relevant.

- [X] FEAT (Add a new feature to the project)

# Checklist:

- [X] My pull request references the issue in the comments
- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [X] I have added tests that prove my fix is effective or that my feature works
- [X]  New and existing unit tests pass locally with my changes

### CODE REVIEW

File: .github/workflows/frontend.yml [hunk 1]
```diff
@@ -93,3 +93,5 @@ jobs:
         args: deploy --only hosting
       env:
         FIREBASE_TOKEN: ${{ secrets.FIREBASE_SERVICE_ACCOUNT_SUPREME_ERP }}
+        PROJECT_PATH: './frontend'
```
Comment (phong1233) [inline .github/workflows/frontend.yml]: I don't think this line is necessary, you can test the deployment by commenting out the if statement in the deploy phase

Comment (tjosepo) [reply .github/workflows/frontend.yml]: This was recommended by the GitHub Action for Firebase documentation.

image

Comment (phong1233) [reply .github/workflows/frontend.yml]: The firebase deployment doesn't use our "repo". We save the files we need in memory then we download it to deploy with firebase. Right now it's set up to download the build folder and the `firebase.json` and `firebaserc`. I think it downloads those files to root

**Review by phong1233 — APPROVED** [review]

**Review by sebastien-blain — APPROVED** [review]
Finally, looks awesome.

**Pull request merged**